src/main/java/uk/gov/hmcts/reform/cdapi/config/LaunchDarklyConfiguration.java

```diff
@@ -1,21 +1,13 @@
 package uk.gov.hmcts.reform.cdapi.config;
 
-import com.launchdarkly.sdk.server.LDClient;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @Configuration
 public class LaunchDarklyConfiguration implements WebMvcConfigurer {
 
-    @Bean
-    public LDClient ldClient(@Value("${launchdarkly.sdk.key}") String sdkKey) {
-        return new LDClient(sdkKey);
```

Comment:
Is this replaced with something else, surely we'll lose access to feature flags without this?

Reply:
uk.gov.hmcts.reform.cdapi.config.LaunchDarklyClientConfig added in this class